api/util/patron.py

```diff
@@ -64,7 +65,8 @@ def assert_borrowing_privileges(cls, patron):
 
         if patron.fines:
             max_fines = Configuration.max_outstanding_fines(patron.library)
-            if max_fines is not None and patron.fines > max_fines.amount:
+            fines = MoneyUtility.parse(patron.fines)
+            if max_fines is not None and fines.amount > max_fines.amount:
```

Comment:
This is a bug caught by the 3.0 conversion -- we were comparing a string (patron.fines) to a number (max_fines.amount).